src/Robo/Plugin/Traits/DatabaseDownloadTrait.php

```diff
@@ -55,7 +55,7 @@ public function databaseDownload(string $siteName = 'default')
         // Ensure objects are sorted by last modified date.
         usort($objects, fn($a, $b) => $a->getLastModified()->getTimestamp() <=> $b->getLastModified()->getTimestamp());
         $latestDatabaseDump = array_pop($objects);
-        $dbFilename = $latestDatabaseDump->getKey();
+        $dbFilename = $this->sanitizeFileNameForWindows($latestDatabaseDump->getKey());
```

Comment:
Will this break line 65 since we are changing the filename that we send to S3 to tell it what we want to download?

Or what am I not understanding correctly?

Also, if you have tested this successfully and can confirm it's not an issue let me know!

`composer require chromatichq/usher:dev-YOUR-BRANCH-NAME`

Reply:
@adamzimmermann You were absolutely right, I've managed to mess it up in my fork already. 

This version works for me and _should_ leave non-Windows environments alone.

Reply:
> If I would have remembered the correct workflow, I would have struggled finding the correct people/person.

Feel free to add me to any back-end PR's. If there is someone better for a given project I'll point you in the right direction! Also feel free to ask around in Slack.